### ISSUE TITLE
Unable to open Trivy generated sarif file

### ISSUE THREAD

**Issue by uandrei** [microsoft/sarif-vscode-extension#437]
Trying to open a Trivy generated SARIF file and got an error that version 2.1.0 and schema https://json.schemastore.org/sarif-2.1.0-rtm.5.json are not supported. Updated the schema URL to https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json and all i get is a panel saying 26 Sarif results in the header but no content.

Extension host logs shows no errors or anything related to the Sarif extension.

VS Code version: v1.67.2
Sarif viewer: v3.1.1

**Comment by jeffersonking** [comment]
Hi @uandrei. How consistently are you getting the "26 Sarif results in the header but no content" as you close and re-open the file? 100% of the time?

**Comment by uandrei** [comment]
@jeffersonking yes this is 100% reproducible as it never worked for me on VS Code. 

This is the command i use on my WSL2 ubuntu to run to generate a sarif file: 
`trivy image -f sarif mcr.microsoft.com/dotnet/runtime:6.0-jammy`

Here's an archive of the sarif file generated.
[repo.zip](https://github.com/microsoft/sarif-vscode-extension/files/8785924/repo.zip)

This image shows the error being thrown when using ` https://json.schemastore.org/sarif-2.1.0-rtm.5.json` schema:
![sarif-error](https://user-images.githubusercontent.com/11975434/170681511-86bdfca6-e2d6-4c92-b438-a858e8751bfa.png)

This shows the screen when using the `https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json` schema
![sarif-error-2ndschema](https://user-images.githubusercontent.com/11975434/170681517-6fffc408-8421-43a5-99ef-0e1d54789a39.png)

At the moment i use https://microsoft.github.io/sarif-web-component/ for going through the sarif files, but would love something local tbh which is why i was looking at this extension.

**Comment by pig837** [comment]
I think that this is a bug.

You can use an workaound.

1. Open .sarif file using notepad. (generated by binskim v2.0.0-rc1)
2. Modify line 2
   - From.   "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.6.json",
   - To.   "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
4. Open .sarif file using vscode (Sarif Viewer v3.3.5)
5. Profit!
